app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,11 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+
+    <color name="color_gray_white">#a8a8a8</color>
+    <color name="color_orange">#FF5500</color>
+    <color name="color_gray">#666666</color>
+    <color name="color_gray_dark">#333333</color>
+    <color name="color_white">#FFFFFF</color>
+    <color name="color_gray_b">#BBBBBB</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_gray_b` appears to be unused
 Unused resources make applications larger and slow down builds.